packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx

```diff
@@ -83,6 +83,13 @@ class CollapsibleMenuViewItem extends React.Component {
     super(props);
     this.setButtonNode = this.setButtonNode.bind(this);
     this.getButtonNode = this.getButtonNode.bind(this);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+  }
+
+  handleButtonClick(event) {
+    if (this.props.onClick) {
```

Comment:
Since this function is actually doing anything, why don't we just remove this line: https://github.com/cerner/terra-framework/pull/1425/files#diff-73b8965c3503d92ed6bbad70f722b2db46bf3e9d58183ff7e5417ad4cc419bdfR153
the onClick prop would just get passed in through the attributes. No need to wrap it.

Reply:
Done